src/modules/dashboard/components/Dialogs/EditExpenditureDialog/ChangedMultiple/ChangedMultiple.css

```diff
@@ -0,0 +1,33 @@
+.headerStyles {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+  letter-spacing: var(--spacing-medium);
+}
+
+.header {
+  composes: headerStyles;
+  margin-top: 20px;
+}
+
+.subheader {
+  composes: headerStyles;
+  display: flex;
+  justify-content: space-between;
+  padding: 6px 0;
+}
+
+.buttonWrappper {
+  display: flex;
+  justify-content: end;
```

Comment:
The use of `end` is giving me warnings in the compiler. It recommends `flex-end` instead.

![end](https://user-images.githubusercontent.com/64402732/189318559-c2c0f6fc-e268-4777-8c9e-1e7dee4b24b9.png)